logging_demo/README.md

```diff
@@ -0,0 +1,75 @@
+# Logging and logger configuration
+
+See [the logging page]() for details on available functionality.
```

Comment:
Needs a link?

Reply:
yeah I'll add it when I move it to the wiki 👍 sorry it wasn't clear that this is temporary.